src/herbpy/tsr/pitcher.py

```diff
@@ -1,9 +1,10 @@
-import numpy
+import numpy, openravepy
+import prpy.tsr
 from prpy.tsr.tsrlibrary import TSRFactory
 from prpy.tsr.tsr import TSR, TSRChain
-
-@TSRFactory('herb', 'rubbermaid_ice_guard_pitcher', 'grasp')
-def pitcher_grasp(robot, pitcher, manip=None):
+      
+@prpy.tsr.tsrlibrary.TSRFactory('herb', 'rubbermaid_ice_guard_pitcher', 'push_grasp')
```

Comment:
You already `import`ed `TSRFactory`, so this full-qualification is not necessary.